### DISCUSSION TITLE
How to lint a subfolder?

### DISCUSSION THREAD

**Discussion by stokito** [super-linter/super-linter#5433] [Q&A]
I have a Go project that has `/web/` folder with html UI.
Now I want to configure a CSS linter and created `/web/.stylelintrc.json` file.
But it's ignored. As far I understood the linter will load the config only if it's located in root `/.stylelintrc.json`.
So how can I configure the CSS linter to check only the `/web/` folder and look for a config inside it?

**Comment by ferrarimarco** [comment]
The README has some details about your use case. Can you share your current configuration, so we can see what's going on?
